include/flamegpu/simulation/detail/SimLogger.h

```diff
@@ -34,7 +34,7 @@ class SimLogger {
      * @param _export_step_time If true step log time will be exported
      * @param _export_exit_time If true exit log time will be exported
      */
-    SimLogger(const std::vector<RunLog> &run_logs,
+    SimLogger(const std::map<unsigned int, RunLog> &run_logs,
```

Comment:
This might be enough of an API break it will need mentioning in the changelog?

Reply:
This is internal (`detail`).